pkg/apis/extensions/v1beta1/types.go

```diff
@@ -966,6 +966,11 @@ type PodSecurityPolicySpec struct {
 	// If set to false the container may run with a read only root file system if it wishes but it
 	// will not be forced to.
 	ReadOnlyRootFilesystem bool `json:"readOnlyRootFilesystem,omitempty" protobuf:"varint,14,opt,name=readOnlyRootFilesystem"`
+	// SeccompProfiles is a slice of allowed profiles that may be set for the pod or
+	// container's seccomp annotations.  The wildcard '*' may be used to allow all.  When
+	// used to generate a value for a pod the first non-wildcard profile will be used as
+	// the default.
+	SeccompProfiles []string `json:"seccompProfiles,omitempty" protobuf:"bytes,15,opt,name=seccompProfiles"`
```

Comment:
figured I'd take the next number since sysctl was still in WIP.  I'll update if it merges first